contents/handbook/company/culture.md

```diff
@@ -6,30 +6,28 @@ showTitle: true
 
 So, what's it like working at PostHog?
 
-<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/rRwzJiljpSA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+## We're 100% remote 
 
-## All remote
+[Our team](/people) is 100% remote, and distributed across more than 15 countries.
 
-Our [team](/people) is 100% remote, and distributed across over 10 countries.
+In addition to all the equipment you'll need, we provide [a budget to help you find coworking space](/handbook/people/spending-money#work-space), or to cover the costs of coffees for those who prefer not to work at home.
```

Comment:
I think this should be below the list of advantages.

Reply:
done